src/app/ipv/router.js

```diff
@@ -41,43 +38,6 @@ router.post(
   (req, res, next) =>
     handleUpdateNameDobAction(req, res, next, "update-name-date-birth"),
 );
```

Comment:
This seems like quite a big change that isn't really related to this ticket and doesn't appear to keep exactly the same behaviour.
That worries me that either the change in behaviour is accidental, or this should be it's own ticket with related testing requirements.

Reply:
This was intentional refactoring whilst I was dealing with the dilemma of creating a new handler - turns out a lot of the handlers aren't needed and so I thought I could include a simplifying refactor into this PR